src/Store/QuickmenuStoreConnector.re

```diff
@@ -49,12 +49,7 @@ let start = (themeInfo: Model.ThemeInfo.t) => {
     });
 
   let makeBufferCommands = (languageInfo, iconTheme, buffers) => {
-    let currentDirectory = Rench.Environment.getWorkingDirectory();
-
-    let getDisplayPath = (fullPath, dir) => {
-      let re = Str.regexp_string(dir ++ Filename.dir_sep);
-      Str.replace_first(re, "", fullPath);
-    };
+    let currentDirectory = Rench.Environment.getWorkingDirectory(); // TODO: This should be workspace-relative
```

Comment:
I found a few of these, which are going to break functionality when you `:cd`

Reply:
Ah, good catch